### PR TITLE
Adding platform element under featureManager

### PR DESCRIPTION
**Pull request by cbridgha**

### CODE REVIEW

**Review by hlhoots — DISMISSED** [review]
LGTM

File: modules/reference/pages/config/featureManager.adoc [hunk 1]
```diff
@@ -9,6 +9,15 @@
...
 |Name|Type|Default|Description
 
+|+platform+
+
+|string +
+This is specified as a child element rather than as an XML attribute.
+
+|
+
+|+The platform specifies a programming model version; for example "jakartaee-10". The platform determines the version of features for which no version is specified.+
```
Comment (dmuelle) [inline modules/reference/pages/config/featureManager.adoc]:
```suggestion
|+The platform specifies a programming model version, for example jakartaee-10. The platform version determines the version of for any enabled versionless features that are included in the specified platform.+
```

Comment (cbridgha) [reply modules/reference/pages/config/featureManager.adoc]: This was copied directly from metatype.xml/property files.  so I can change it here but need to keep coordinated in source as well....

File: modules/reference/pages/config/featureManager.adoc [hunk 2]
```diff
@@ -9,6 +9,15 @@
...
 |Name|Type|Default|Description
 
+|+platform+
+
+|string +
+This is specified as a child element rather than as an XML attribute.
+
+|
+
+|+The platform specifies a programming model version, for example jakartaee-10. The platform version determines the version of for any enabled versionless features that are included in the specified platform.+
```
Comment (dmuelle) [inline modules/reference/pages/config/featureManager.adoc]:
```suggestion
|+The platform specifies a programming model version, for example jakartaee-10. The platform version determines the version of any enabled versionless features that are included in the specified platform.+
```

**Review by dmuelle — APPROVED** [review]

**Pull request merged**